english/challenge-marshmellow-to-rock-off/castle-route/ladder/matrix.md

```diff
@@ -0,0 +1,12 @@
+You climb up the ladder, and reach what looks like a manhole cover at the top.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.